.github/workflows/rust.yml

```diff
@@ -1,42 +1,89 @@
-name: RocksDB build
+name: RocksDB CI
 
 on: [push, pull_request]
 
 jobs:
-  test-librocksdb-sys:
-    name: Test librocksdb-sys crate
-    runs-on:
-      - windows-latest
+  fmt:
+    name: Rustfmt
+    runs-on: ubuntu-latest
     steps:
       - name: Checkout sources
         uses: actions/checkout@v2
+      - name: Install rust
+        uses: actions-rs/toolchain@v1
         with:
-          submodules: true
+          toolchain: stable
+          components: rustfmt
+          profile: minimal
+          override: true
+      - name: Run rustfmt
+        uses: actions-rs/cargo@v1
+        with:
+          command: fmt
+          args: --all -- --check
 
-      - name: Install dependencies
-        run: choco install llvm -y
+  clippy:
+    name: Clippy
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+      - name: Install rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          components: clippy
+          profile: minimal
+          override: true
+      - name: Run clippy
+        uses: actions-rs/clippy-check@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Just out of curiosity. Is that token really needed ?

Reply:
According to the documentation, it is required: https://github.com/actions-rs/clippy-check#inputs